src/Sylius/Component/Shipping/Checker/ShippingMethodEligibilityChecker.php

```diff
@@ -13,11 +13,14 @@
 
 namespace Sylius\Component\Shipping\Checker;
 
-use const E_USER_DEPRECATED;
 use Sylius\Component\Shipping\Model\ShippingMethodInterface;
 use Sylius\Component\Shipping\Model\ShippingSubjectInterface;
 
-@trigger_error(sprintf('The "%s" class is deprecated since Sylius 1.8, use "%s" instead.', 'Sylius\Component\Shipping\Checker\ShippingMethodEligibilityChecker', 'Sylius\Component\Shipping\Checker\Eligibility\CompositeShippingMethodEligibilityChecker'), E_USER_DEPRECATED);
+trigger_deprecation(
+    'sylius/shipping',
+    '1.8',
+    sprintf('The "%s" class is deprecated, use "%s" instead.', 'Sylius\Component\Shipping\Checker\ShippingMethodEligibilityChecker', 'Sylius\Component\Shipping\Checker\Eligibility\CompositeShippingMethodEligibilityChecker'),
+);
```

Comment:
```suggestion
trigger_deprecation(
    'sylius/shipping',
    '1.8',
    'The "%s" class is deprecated, use "%s" instead.', 	 
    ShippingMethodEligibilityChecker::class,   
    CompositeShippingMethodEligibilityChecker::class,
);
```

`sprintf` is not needed, as 4th+ arguments are used as a `printf` arguments (read the `trigger_deprecation` docblock.